lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureBlockBlobFileSystem.cs

```diff
@@ -162,13 +162,27 @@ public void DeleteDirectory(Uri directoryUri)
         public IEnumerable<Uri> GetChildren(Uri directoryUri)
         {
             BlobContinuationToken blobContinuationToken = null;
-            var path = directoryUri.AbsolutePath.Trim('/');
+            string path = directoryUri.AbsolutePath.Trim('/');
```

Comment:
This path work is unnecessary. Just use directoryUri.Segments

Reply:
Done. Thanks for the suggestion.